sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -139,22 +145,31 @@ def list_configuration_settings(
         *,
         key_filter: Optional[str] = None,
         label_filter: Optional[str] = None,
+        tags_filter: Optional[List[str]] = None,
         accept_datetime: Optional[Union[datetime, str]] = None,
-        fields: Optional[List[str]] = None,
+        fields: Optional[List[Union[str, ConfigurationSettingFields]]] = None,
         **kwargs: Any,
     ) -> ItemPaged[ConfigurationSetting]:
         """List the configuration settings stored in the configuration service, optionally filtered by
         key, label and accept_datetime.
 
-        :keyword key_filter: filter results based on their keys. '*' can be
-            used as wildcard in the beginning or end of the filter
+        :keyword key_filter: Filter results based on their keys. '*' can be used as wildcard in the beginning or end
+            of the filter. For more information about supported filters,
+            see https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#supported-filters.
         :paramtype key_filter: str or None
-        :keyword label_filter: filter results based on their label. '*' can be
-            used as wildcard in the beginning or end of the filter
+        :keyword label_filter: Filter results based on their label. '*' can be used as wildcard in the beginning or end
+            of the filter. For more information about supported filters,
+            see https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#supported-filters.
         :paramtype label_filter: str or None
-        :keyword accept_datetime: retrieve ConfigurationSetting existed at this datetime
+        :keyword tags_filter: A filter used to query by tags. Default value is None.
+            For more information about supported filters,
+            see https://learn.microsoft.com/en-us/azure/azure-app-configuration/rest-api-key-value#supported-filters.
```

Comment:
Please remove `/en-us/` from all URLs

Reply:
This URL doesn't have any information on supported tag filters, so it's not helpful to reference it here.

Reply:
Maybe the portal hasn't updated the new filter yet, I'm agree to remove the reference for now.

Reply:
The tags filter is in the latest version pivot: https://learn.microsoft.com/azure/azure-app-configuration/rest-api-revisions?pivots=v23-11#supported-filters